lib/clients/zookeeper.js

```diff
@@ -49,6 +50,16 @@ function createClient(connectionString, options) {
             return zkClient.emit('ready');
         });
     });
+    zkClient.rmr = function rmr(path, cb) {
```

Comment:
it's pretty unclear what `rmr` stands for, I'm guessing 'remover' or something like that, but maybe you could make the function name a little more obvious?

Reply:
It means recursive removal. It's actually the command name in zkCli.sh so I wanted to map against it, but for a function name I can make it more descriptive like `deleteRecur`